frontend/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 120
+}
```

Comment:
Should these 3 `frontend/.*` files be included in .gitignore?

Reply:
Great questions, these come from https://github.com/patternfly/patternfly-react-seed which I used as a base. I did not remove anything used by them I just built on top of it.

Reply:
It looks like they are not essential to the code itself. I don't have a preference on removing them or not.

Reply:
Updated the gitignore and removed these files.